src/Infrastructure/Infrastructure.csproj

```diff
@@ -1,19 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
-        <RootNamespace>Codidact.Authentication.Infrastructure</RootNamespace>
-        <AssemblyName>Codidact.Authentication.Infrastructure</AssemblyName>
-    </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Codidact.Authentication.Infrastructure</RootNamespace>
+    <AssemblyName>Codidact.Authentication.Infrastructure</AssemblyName>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.2" />
-        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.2" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.2" />
-        <PackageReference Include="IdentityServer4" Version="3.1.2" />
-        <PackageReference Include="IdentityServer4.AspNetIdentity" Version="3.1.2" />
-    </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.2" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.0" />
```

Comment:
The connection string in [docs/defaults/src/WebApp/appsettings.json][1] has to be updated too. Something like

~~~none
Host=localhost;Database=codidact;Username=codidact
~~~

  [1]: https://github.com/misha130/authentication/blob/7b95f8863fc724653f5d60b09d8629beb161cfb9/docs/defaults/src/WebApp/appsettings.json#L32

Reply:
Should I revert it to Sqlite?

Reply:
No, I was about to switch to PostgreSQL myself, in the next pull request. Just update the connection string, that is enough. I was planning to rewrite the documentation and can do this properly then.